plugins/masters-tournament/masters_helpers.py

```
@@ -453,8 +453,13 @@ def get_detailed_phase(
                 return "tournament-live"
             return "tournament-evening"
 
-        if date > end_e and (date - end_e) <= timedelta(days=post_tournament_display_days):
-            return "post-tournament"
+        if date > end_e:
+            # Use calendar-day comparison in EDT so post_tournament_display_days
+            # means N full calendar days, not N×24 hours from the padded end
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)